README.md

```diff
@@ -237,3 +237,41 @@ If you want to grant members of other groups ability to view the results, you ca
 names in the django settings using the `XBLOCK_POLL_EXTRA_VIEW_GROUPS` setting, for example:
 
     XBLOCK_POLL_EXTRA_VIEW_GROUPS = ['poll_staff']
+
+## Working with Translations
+
+The following sections provide basic information about common use cases for working with translations. You can find additional information in [this section](http://edx.readthedocs.io/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html#adding-translated-strings-to-your-xblock) of the [Open edX XBlock Tutorial](http://edx.readthedocs.io/projects/xblock-tutorial/en/latest/index.html).
+
+### Viewing the XBlock in a Different Language
+
+To view this XBlock in a different language you first need to compile the `text.po` file for the language to a file called `text.mo`.  Translation files for a given language are located in the `poll/translations/<locale>/LC_MESSAGES` directory. From the top-level directory of this repository, use the following commands to access the directory and compile the translation file (replacing `<lang_code>` with the appropriate language code):
+
+```sh
+cd poll/translations/<lang_code>/LC_MESSAGES/
+msgfmt text.po -o text.mo
+```
+
+You can then navigate to a poll or survey in the LMS and view a translated version of it by adding `/?preview-lang=<lang_code>` to the URL. Note that user-editable content (such as questions and answers belonging to a poll) is not translated by default. To revert to the default language, add `/?clear-lang` to the URL.
+
+### Adding Translations for Other Languages
+
+To add translations for other languages, create a set of language directories for each of your locales within the `poll/translations` directory. You may specify either a general language code or a specific language locale code for the name of each directory. Include an `LC_MESSAGES` directory with each language directory. For example:
+
+`../poll/translations/ar/LC_MESSAGES/`
+`../poll/translations/es-es/LC_MESSAGES/`
+
+For each language, create a domain file named `text.po` and compile it: You can use edX's [i18n-tools](https://github.com/edx/i18n-tools) to extract translations for multiple languages as follows:
+
+```sh
+i18n_tool extract
```

Comment:
@asadiqbal08 This command works when I run it from the `edx-platform` directory. But it won't update the translation files for xblock-poll (I changed a string in poll.py beforehand, and it didn't get picked up). When I run the command from the directory where xblock-poll is mounted on my devstack, I get `Exception: Configuration file cannot be found: /edx/app/edxapp/xblocks/xblock-poll/conf/locale/config.yaml`. So it looks like this section needs some additional information about creating a config file?

I assume you have a working config file for extracting strings for xblock-poll? It would be worth adding the contents of that file as an example here.